doc/user/known-issues.txt

```diff
@@ -0,0 +1,26 @@
+.. vim: set filetype=rst
+
+============
+Known Issues
+============
+
+This document details the known issues in the current release of screed. All
+issues for screed are tracked at https://github.com/ged-lab/khmer/issues
```

Comment:
I would link directly to the screed issues at https://github.com/ged-lab/khmer/issues?q=label%3Ascreed